target/linux/x86/legacy/config-6.1

```diff
@@ -229,34 +419,100 @@ CONFIG_SERIAL_8250_PNP=y
 # CONFIG_SND_SOC_AMD_PS is not set
 # CONFIG_SND_SOC_AMD_RPL_ACP6x is not set
 # CONFIG_SND_SOC_INTEL_AVS is not set
+CONFIG_SOCK_RX_QUEUE_MAPPING=y
 # CONFIG_SURFACE_PLATFORMS is not set
+CONFIG_SWIOTLB=y
 CONFIG_SYNC_FILE=y
 # CONFIG_SYSTEM76_ACPI is not set
+CONFIG_SYS_HYPERVISOR=y
+CONFIG_TASKSTATS=y
+CONFIG_TASK_DELAY_ACCT=y
+CONFIG_THERMAL_GOV_USER_SPACE=y
+CONFIG_THERMAL_HWMON=y
+CONFIG_THERMAL_WRITABLE_TRIPS=y
 # CONFIG_THINKPAD_LMI is not set
 # CONFIG_TOSHIBA_BT_RFKILL is not set
 # CONFIG_TOSHIBA_WMI is not set
+CONFIG_TREE_RCU=y
+CONFIG_TREE_SRCU=y
+# CONFIG_UCLAMP_TASK is not set
+CONFIG_UCS2_STRING=y
 CONFIG_USB_STORAGE=y
+CONFIG_USER_RETURN_NOTIFIER=y
+CONFIG_VHOST=y
+CONFIG_VHOST_IOTLB=y
+CONFIG_VHOST_NET=y
 # CONFIG_VIDEO_IPU3_CIO2 is not set
+CONFIG_VIRTIO=y
+CONFIG_VIRTIO_BALLOON=y
+CONFIG_VIRTIO_BLK=y
+CONFIG_VIRTIO_CONSOLE=y
+CONFIG_VIRTIO_DMA_SHARED_BUFFER=y
+CONFIG_VIRTIO_INPUT=y
+CONFIG_VIRTIO_MMIO=y
+CONFIG_VIRTIO_NET=y
+CONFIG_VIRTIO_PCI=y
+CONFIG_VIRTIO_PCI_LEGACY=y
+CONFIG_VIRTIO_PCI_LIB=y
+# CONFIG_VIRTIO_PMEM is not set
+CONFIG_VIRTUALIZATION=y
 CONFIG_VMAP_PFN=y
+CONFIG_VT_CONSOLE_SLEEP=y
+CONFIG_WATCHDOG_CORE=y
 # CONFIG_WDT is not set
 # CONFIG_WIRELESS_HOTKEY is not set
 # CONFIG_WMI_BMOF is not set
+CONFIG_X86_32_SMP=y
 CONFIG_X86_ACPI_CPUFREQ=y
 # CONFIG_X86_ACPI_CPUFREQ_CPB is not set
 CONFIG_X86_ALIGNMENT_16=y
-# CONFIG_X86_AMD_FREQ_SENSITIVITY is not set
-# CONFIG_X86_AMD_PLATFORM_DEVICE is not set
-# CONFIG_X86_AMD_PSTATE is not set
+CONFIG_X86_AMD_FREQ_SENSITIVITY=y
+CONFIG_X86_AMD_PLATFORM_DEVICE=y
+CONFIG_X86_AMD_PSTATE=y
 # CONFIG_X86_AMD_PSTATE_UT is not set
+# CONFIG_X86_BIGSMP is not set
+CONFIG_X86_CPUID=y
 # CONFIG_X86_E_POWERSAVER is not set
 CONFIG_X86_F00F_BUG=y
-# CONFIG_X86_INTEL_LPSS is not set
+CONFIG_X86_HV_CALLBACK_VECTOR=y
+CONFIG_X86_INTEL_LPSS=y
+CONFIG_X86_INTEL_PSTATE=y
 # CONFIG_X86_LONGHAUL is not set
 CONFIG_X86_MINIMUM_CPU_FAMILY=5
+CONFIG_X86_NEED_RELOCS=y
 # CONFIG_X86_PAE is not set
 # CONFIG_X86_PCC_CPUFREQ is not set
+CONFIG_X86_PKG_TEMP_THERMAL=y
+# CONFIG_X86_PMEM_LEGACY is not set
 CONFIG_X86_PM_TIMER=y
 # CONFIG_X86_POWERNOW_K8 is not set
+CONFIG_XEN=y
+CONFIG_XENFS=y
+CONFIG_XEN_ACPI=y
+CONFIG_XEN_AUTO_XLATE=y
+# CONFIG_XEN_BACKEND is not set
+CONFIG_XEN_BALLOON=y
+CONFIG_XEN_BLKDEV_FRONTEND=y
+CONFIG_XEN_COMPAT_XENFS=y
+CONFIG_XEN_DEBUG_FS=y
+CONFIG_XEN_DEV_EVTCHN=y
+CONFIG_XEN_FBDEV_FRONTEND=y
+CONFIG_XEN_GNTDEV=y
+CONFIG_XEN_GRANT_DEV_ALLOC=y
+CONFIG_XEN_NETDEV_FRONTEND=y
+CONFIG_XEN_PRIVCMD=y
+CONFIG_XEN_PVH=y
+CONFIG_XEN_PVHVM=y
+CONFIG_XEN_PVHVM_GUEST=y
+CONFIG_XEN_PVHVM_SMP=y
+CONFIG_XEN_SAVE_RESTORE=y
+CONFIG_XEN_SCSI_FRONTEND=y
+CONFIG_XEN_SYS_HYPERVISOR=y
+CONFIG_XEN_VIRTIO=y
+# CONFIG_XEN_VIRTIO_FORCE_GRANT is not set
+CONFIG_XEN_WDT=y
+CONFIG_XEN_XENBUS_FRONTEND=y
```

Comment:
Of all the hypervisors, Xen was originally written to function on `legacy`-class hardware.  The support may die off soon if it hasn't already been killed, but it _did_ work.